openedx/features/partners/models.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import User
+from django.db import models
+from model_utils.models import TimeStampedModel
+
+
+class Partner(TimeStampedModel):
+    """
+    This model represents white-labelled partners.
+    """
+    label = models.CharField(max_length=100, default=None)
+    main_logo = models.CharField(max_length=255, default=None)
+    small_logo = models.CharField(max_length=255, default=None)
+    slug = models.CharField(max_length=100, default=None, unique=True)
+
```

Comment:
all the fields has `null=False` by default, So `default=None` can be removed in all fields.